packages/nc-gui/components/cell/Email.vue

```diff
@@ -74,9 +74,9 @@ watch(
     :ref="focus"
     v-model="vModel"
     :placeholder="isEditColumn ? '(Optional)' : ''"
-    class="w-full focus:!outline-none focus:ring-0 focus:border-none !border-none !outline-none text-sm px-2"
+    class="w-full focus:ring-0 focus:border-none border-none outline-none text-sm px-2"
```

Comment:
`focus:border-none` should be not necessary. please double check.

Reply:
ok